Privacy-policy.md

```diff
@@ -0,0 +1,17 @@
+# Privacy Policy
+
+## Effective date: September 06, 2018
+
+[Codemancers](https://codemancers.com) operates the Interceptor browser extension. ([Chrome](https://chrome.google.com/webstore/detail/interceptor/enenfaicdcfgcnjmiigcjbmlbaoapnen) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/xhr-interceptor/)).
+
+Interceptor browser extension does not collect or share personal information of any kind. That is our privacy policy.
```

Comment:
You can remove "That is our privacy policy". You can replace that with - "Interceptor does not share any data accepted from the users with third parties."